$OpenBSD: patch-configure,v 1.1 2010/10/01 08:14:15 jasper Exp $

- Fix appending to shell variables.
- usnet is not a valid shell command.

--- configure.orig	Thu Sep 30 22:36:13 2010
+++ configure	Thu Sep 30 22:37:39 2010
@@ -16440,8 +16440,8 @@ else
 fi
 
 if test x$disable_deprecated = xyes; then
-	CFLAGS+=" -DG_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES"
-	CFLAGS+=" -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
+	CFLAGS="$CFLAGS -DG_DISABLE_SINGLE_INCLUDES -DGDK_PIXBUF_DISABLE_SINGLE_INCLUDES -DGTK_DISABLE_SINGLE_INCLUDES"
+	CFLAGS="$CFLAGS -DG_DISABLE_DEPRECATED -DGDK_DISABLE_DEPRECATED -DGDK_PIXBUF_DISABLE_DEPRECATED -DGTK_DISABLE_DEPRECATED"
 fi
 
 
@@ -16460,7 +16460,7 @@ if test x$ENABLE_DEBUG = xyes; then
 
 $as_echo "#define DEBUG 1" >>confdefs.h
 
-	CFLAGS+=" -g -O0 -DDEBUG"
+	CFLAGS="$CFLAGS -g -O0 -DDEBUG"
 fi
 
 # Check whether --enable-gseal was given.
@@ -16475,7 +16475,7 @@ else
 fi
 
 if test x$ENABLE_GSEAL = xyes; then
-	CFLAGS+=" -DGSEAL_ENABLE"
+	CFLAGS="$CFLAGS -DGSEAL_ENABLE"
 fi
 
 # Check whether --enable-run_in_place was given.
@@ -18769,7 +18769,7 @@ $as_echo "#define HAVE_LIBJPEG 1" >>confdefs.h
 		JPEG_LIBS='-ljpeg -lm -lz'
 	fi
 	CFLAGS="$SAVE_CFLAGS"
-	usnet SAVE_CFLAGS
+	unset SAVE_CFLAGS
 fi
 
  if test "x$enable_jpeg" = xyes; then
